_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,88 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or `area` elements, or any [non-decorative](#decorative) HTML element with the semantic role of `img`, where the element is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute. Difference in letter casing, and forward and trailing whitespace should be ignored. 
```

Comment:
`area` can also be decorative. 

But `area` has the semantic role of `link` not `img`so it can't be included in the "or any [non-decorative](#decorative) HTML element with the semantic role of `img`".

I suggest to reorder and to split to make it easier to read  : 

> The rule applies to `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, `area` element, or any HTML element with the semantic role of `img`, where the element is [non-decorative](#decorative) and is [included in the accessibility tree](#included-in-the-accessibility-tree).
The element has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute. Difference in letter casing, and forward and trailing whitespace should be ignored. 


**Note** : I changed "elements" to the singular "element" (other rule use the singular for the applicability "the X element")

Reply:
Updated to include decorative `area` elements
Updated to use the singular "element" instead of the plural "elements".